config/v1/types_feature.go

```diff
@@ -29,12 +29,40 @@ var (
 	// TechPreviewNoUpgrade turns on tech preview features that are not part of the normal supported platform. Turning
 	// this feature set on CANNOT BE UNDONE and PREVENTS UPGRADES.
 	TechPreviewNoUpgrade FeatureSet = "TechPreviewNoUpgrade"
+
```

Comment:
Is there ever a situation where we want a TechPrevewUpgrade to be defined?

Reply:
> Is there ever a situation where we want a TechPrevewUpgrade to be defined?

Generically, it's very dangerous as some features graduate and some die.  We chose this name in 4.1 so that no one could say, "I expected you to upgrade".  Then of course, we have someone upset that they can't upgrade...

Reply:
Tech preview implies documentation and support training across the OKD community.

It is unlikely we would have a TechPreviewUpgrade, but its possible a scenario comes up in the future where that makes sense (it just hasnt happened thus far).